src/select/select.tsx

```diff
@@ -40,11 +41,12 @@ class Select<T> extends React.Component<Props<T>, State<T>> {
         this.setNode = this.setNode.bind(this);
         this.onKeyPressed = this.onKeyPressed.bind(this);
         this.filterOptions = this.filterOptions.bind(this);
-        this.toggleOptionsIfNotDisabled = this.toggleOptionsIfNotDisabled.bind(this);
+        this.isOptionDisabled = this.isOptionDisabled.bind(this);
         this.handleGlobalClick = this.handleGlobalClick.bind(this);
         this.handleOptionClick = this.handleOptionClick.bind(this);
         this.toggleOptionsList = this.toggleOptionsList.bind(this);
         this.collapseOptionsList = this.collapseOptionsList.bind(this);
+        this.toggleOptionsIfNotDisabled = this.toggleOptionsIfNotDisabled.bind(this);
     }
 
     public componentWillMount() {
```

Comment:
Not strictly related to your PR, but could we change this to `componentDidMount` since `componentWillMount` is being deprecated ?